resources/app.yaml.tpl

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
```

Comment:
`VERSION` does not have to be template, `app import` has `--version` flag that sets the version for the imported application